src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor

```diff
@@ -0,0 +1,36 @@
+﻿@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Otlp.Model
+@using Aspire.Dashboard.Resources
+@inject IStringLocalizer<Columns> Loc
+
+<div class="resource-state-container">
+    @if (Resource is { State: ResourceStates.ExitedState /* containers */ or ResourceStates.FinishedState /* executables */, ExitCode: not null and not 0 })
+    {
+        var iconId = $"state-icon-{Resource.GetHashCode()}";
+        <!-- process completed unexpectedly, hence the non-zero code. this is almost certainly an error, so warn users -->
+        <FluentIcon Id="@iconId" Icon="Icons.Filled.Size16.ErrorCircle" Color="Color.Error" Class="severity-icon"/>
+        <FluentTooltip Anchor="@iconId">
```

Comment:
We're using built-in title instead of fluent tooltip everywhere else. Is there a reason to do it differently here?

Reply:
no good reason other than aesthetics. I don't have any opinion, so I'll change it back.